docs/user-guide/address-security-requirements.md

```diff
@@ -0,0 +1,73 @@
+
+The following required configuration tasks are  performed by your organization's security administrator:
+
+* [Initialize Zowe security configurations](../user-guide/initialize-security-configuration.md)
+* [Perform APF authorization of load libraries](../user-guide/apf-authorize-load-library.md)
+* [Configure the z/OS system for Zowe](../user-guide/configure-zos-system.md/#configure-user-ids-and-groups-for-the-zowe-started-tasks)
+* [Configure address space job naming](../user-guide/configure-zos-system.md/#configure-address-space-job-naming)
+* [Assign security permissions of users](#assign-security-permissions-of-users)
+
+If your Zowe server-side installation includes the features listed in the following table, consult your organization's security administrator to perform the associated security tasks:
+
+| Feature of a Zowe server-side component  | Configuration Task  | 
+| ---- | ---- | 
+| Use of Top Secret as your security manager | [Configuring multi-user address space (for TSS only)](../user-guide/configure-zos-system.md/#configure-multi-user-address-space-for-tss-only) |
+| High Availability | [Configuring ZWESLSTC to run Zowe high availability instances under ZWESVUSR user ID](../user-guide/configure-zos-system.md/#configure-zweslstc-to-run-zowe-high-availability-instances-under-zwesvusr-user-id) |
+| z/OSMF authentication or onboarding of z/OSMF service | [Granting users permission to access z/OSMF](../user-guide/grant-user-permission-zosmf.md) |
+| ZSS component enabled (required for API ML certificate and identity mapping) |[Configuring an ICSF cryptographic services environment](../user-guide/configure-zos-system.md/#configure-an-icsf-cryptographic-services-environment) <br />and<br /> [Configuring security environment switching](../user-guide/configure-zos-system.md/#configure-security-environment-switching)|
+| API Mediation Layer certificate mapping | [Configuring main Zowe server to use client certificate identity mapping](../user-guide/configure-zos-system.md/#configure-main-zowe-server-to-use-client-certificate-identity-mapping) |
+| API Mediation Layer identity mapping | [Configuring main Zowe server to use distributed identity mapping](../user-guide/configure-zos-system.md/#configure-main-zowe-server-to-use-distributed-identity-mapping) |
+| API Mediation Layer Identity Tokens (IDT) | [Configuring signed SAF Identity tokens (IDT)](../user-guide/configure-zos-system.md/#configure-signed-saf-identity-tokens-idt) |
+| Cross memory server (ZIS) | [Configuring the cross memory server for SAF](../user-guide/configure-zos-system.md/#configure-the-cross-memory-server-for-saf)<br />and<br />[Configuring cross memory server load module](../user-guide/configure-xmem-server.md/#load-module)<br />and<br />[Configuring cross-memory server SAF configuration](../user-guide/configure-xmem-server.md/#saf-configuration) |
+
+## Assign security permissions of users
```

Comment:
These are also tasks for sec admin. The title should not be on the same level.

Reply:
resolved